README.rst

```diff
@@ -284,6 +284,10 @@ Choose from the list of available fixers:
                         one blank line after the use
                         statements block.
 
+* **switch_case_colon** [PSR-2]
```

Comment:
would renaming `switch_case_colon` to `switch_case_semicolon_to_colon` be better?